src/core/Akka.Streams.Tests/IO/FileSinkSpec.cs

```diff
@@ -221,10 +222,10 @@ Task<IOResult> Write(List<string> lines, long pos) => Source.From(lines)
                             _materializer);
 
                     var completion1 = Write(_testLines, 0);
-                    await completion1.ShouldCompleteWithin(Remaining);
+                    await completion1.ShouldCompleteWithin(2.Seconds());
```

Comment:
`Remaining` defaults to whatever is in `_expectTimeout` - adjust that value instead, otherwise the `Within` block is worthless.

Reply:
This has not been addressed. Please change `2.seconds()` back to `Remaining`